api/odilo.py

```diff
@@ -229,15 +230,17 @@ def record_info_to_metadata(cls, book, availability):
         language = book.get('language', 'spa')
 
         subjects = []
+        trusted_weight = Classification.TRUSTED_DISTRIBUTOR_WEIGHT
         for subject in book.get('subjects', []):
-            subjects.append(SubjectData(type=Subject.TAG, identifier=subject, weight=100))
+            subjects.append(SubjectData(type=Subject.TAG, identifier=subject, weight=trusted_weight))
 
         for subjectBisacCode in book.get('subjectsBisacCodes', []):
-            subjects.append(SubjectData(type=Subject.BISAC, identifier=subjectBisacCode, weight=100))
+            subjects.append(SubjectData(type=Subject.BISAC, identifier=subjectBisacCode, weight=trusted_weight))
 
         grade_level = book.get('gradeLevel')
         if grade_level:
-            subject = SubjectData(type=Subject.GRADE_LEVEL, identifier=grade_level, weight=10)
+            # TODO: It's not clear why we trust this data less.
```

Comment:
Seems likely this code was copied from another distributor without thinking about it.